web/app/mu-plugins/loader.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Plugin Name: Pantheon MU Plugin Loader
+ * Description: Loads the MU plugins required to run the site
+ * Author: Pantheon
+ * Author URI: https://pantheon.io
+ * Version: 1.0
+ * License: MIT
+ */
+
+if ( ( defined( 'WP_INSTALLING' ) && WP_INSTALLING ) ) {
+	return;
+}
+
+// Add your mu-plugins here.
+$mu_plugins = [
```

Comment:
If someone makes a custom upstream derived from this upstream, what are the odds that they might want to add an mu-plugin? Does anyone ever add an mu-plugin to pantheon-systems/WordPress? Do they have to modify this file?

If we put this file in the upstream, we need a strategy for how users will modify it (if necessary) without causing conflicts (e.g. if Pantheon later adds another mu-plugin). We have the same concern if this file is scaffolded, except the solution might be even more difficult in that instance.

Reply:
I'd say it's reasonably common for engineers to add mu-plugins, and I'm sure customers are doing that while using the `pantheon-systems/WordPress` upstream. They _don't_ need to modify this file - WordPress looks for PHP files right inside the mu-plugins directory, and will load those automatically (meaning there can be multiple loader files in this directory).

We could rename this to `pantheon-loader.php` (or something along those lines) and add a comment at the top instructing customers not to modify. Customers can then add their own PHP loader file to `web/app/mu-plugins` to load whichever mu-plugins they add to their site.

Reply:
If we're keeping this file as a `pantheon-loader` then we should provide a suggestion about how they can add their own loader file. Since this file was built to be generic and widely used, I'd say the comment instructing users to _not use this file_ tells them that they can copy the file as the base for their own loader file.